products-service/src/libs/handlerResolver.ts

```diff
@@ -0,0 +1,3 @@
+export const handlerPath = (context: string) => {
+  return `${context.split(process.cwd())[1].substring(1).replace(/\\/g, '/')}`
```

Comment:
I'd cover this function with unit tests, because it looks quite complex :)

Reply:
I've just added single unit test, which demonstrate how it works. Since this method is pretty simple in a base and provided by template, I don't think we need negative tests :)